package/MDAnalysis/units.py

```diff
@@ -43,6 +43,7 @@
    =========== ============== ===============================================
    length       Å              :math:`10^{-10}` m
    time         ps             :math:`10^{-12}` s
+   mass         u              :math:`1.660538921 \times 10^{-27}` kg
```

Comment:
This where is belongs to. Thanks!

 However, I am not sure about the exact value of u. There was had a [comment](https://github.com/MDAnalysis/mdanalysis/issues/3944#issuecomment-1351805481) about a more accurate value. @v-parmar can you give a reference where we are using your value?